boards/ezurio/mg100/doc/index.rst

```diff
@@ -204,10 +204,9 @@ Segger IC.
 Software
 ********
 
-MG100 Out-of-Box Demo Software
+Canvas Software Suite
 ==============================
-The MG100 ships with an out of the box software demo.
-Check out the `BLE Gateway OOB Demo`_ source code and documentation.
+The MG100 is a supported hardware platform for `Canvas Software Suite`_.
```

Comment:
Not sure what this has to do with zephyr, or renaming a vendor

Reply:
This has to do with renaming links at the bottom. Leaving outdated links didn't make sense so I changed the associated description around the link change.